bindings/python/pydeck/docs/layer.md

```diff
@@ -9,30 +9,71 @@ class pydeck.Layer(
     type,
     data,
     id=None,
-    get_position='[lng,lat]',
+    get_position='[lng, lat]',
     **kwargs)
 ```
 
-Constructs a geospatial layer for plotting.
+Constructs a visualization layer from data for rendering on a map.
 
 A catalog of available layers is viewable [here](https://github.com/uber/deck.gl/tree/master/docs/layers#deckgl-layer-catalog-overview).
+
 Note that this is Javascript documentation and parameters in pydeck usually will be snake-cased according to Python convention.
 
+**Also note that parameters change by layer**. Not all layers have all parameters. `get_position`, for instance, is on `ScatterplotLayer` but not `ArcLayer`. Be sure to refer to the deck.gl layer catalog.
+
 #### Parameters
 
 `type` : `str`
-    Type of layer to render, e.g., `HexagonLayer`. See the layer catalog above.
+
+Type of layer to render, e.g., `HexagonLayer`. See the [layer catalog](https://github.com/uber/deck.gl/tree/master/docs/layers#deckgl-layer-catalog-overview).
```

Comment:
The props documented in the layer docs are not in snake case. It is not explained in this document.